app/src/main/java/com/stevesoltys/seedvault/transport/backup/BackupCoordinator.kt

```diff
@@ -143,12 +147,26 @@ internal class BackupCoordinator(
         @Suppress("UNUSED_PARAMETER") isFullBackup: Boolean,
     ): Boolean {
         val packageName = targetPackage.packageName
+        val applicationInfo = targetPackage.applicationInfo
+        // This must be kept in sync with frameworks/base/services/backup/java/com/android/server/backup/utils/BackupEligibilityRules.java
```

Comment:
It could be moved to a method.

Reply:
Why do we need to re-add these rules from AOSP anyway? Would D2D otherwise backup stuff that for some reason we don't want in our backups?

Reply:
https://github.com/seedvault-app/seedvault/pull/407/files/396d5792b8f31ecc8bf175f3f54a14930748dc9f#r875738119 see my reply

Reply:
If there's really no way around copying that logic from AOSP, then please let's do it in a similarly readable fashion.